docs/sources/flow/reference/components/discovery.linode.md

```diff
@@ -0,0 +1,162 @@
+
+* `__meta_linode_instance_id`: the id of the linode instance
+* `__meta_linode_instance_label`: the label of the linode instance
+* `__meta_linode_image`: the slug of the linode instance's image
+* `__meta_linode_private_ipv4`: the private IPv4 of the linode instance
+* `__meta_linode_public_ipv4`: the public IPv4 of the linode instance
+* `__meta_linode_public_ipv6`: the public IPv6 of the linode instance
+* `__meta_linode_region`: the region of the linode instance
+* `__meta_linode_type`: the type of the linode instance
+* `__meta_linode_status`: the status of the linode instance
+* `__meta_linode_tags`: a list of tags of the linode instance joined by the tag separator
+* `__meta_linode_group`: the display group a linode instance is a member of
+* `__meta_linode_hypervisor`: the virtualization software powering the linode instance
+* `__meta_linode_backups`: the backup service status of the linode instance
+* `__meta_linode_specs_disk_bytes`: the amount of storage space the linode instance has access to
+* `__meta_linode_specs_memory_bytes`: the amount of RAM the linode instance has access to
+* `__meta_linode_specs_vcpus`: the number of VCPUS this linode has access to
+* `__meta_linode_specs_transfer_bytes`: the amount of network transfer the linode instance is allotted each month
+* `__meta_linode_extra_ips`: a list of all extra IPv4 addresses assigned to the linode instance joined by the tag separator
+
+## Component health
+
+`discovery.linode` is only reported as unhealthy when given an invalid
+configuration. In those cases, exported fields retain their last healthy
+values.
+
+## Debug information
+
+`discovery.linode` does not expose any component-specific debug information.
+
+### Debug metrics
+
+`discovery.linode` does not expose any component-specific debug metrics.
+
+## Example
+
+```river
+discovery.linode "example" {
+    bearer_token = env("LINODE_TOKEN")
+    port = 8876
+}
+prometheus.scrape "demo" {
+	targets    = discovery.linode.example.targets
+	forward_to = [prometheus.remote_write.demo.receiver]
+}
+prometheus.remote_write "demo" {
+	endpoint {
+		url = PROMETHEUS_REMOTE_WRITE_URL
+		basic_auth {
+			username = USERNAME
+			password = PASSWORD
+		}
+	}
+}
+```
+Replace the following:
+  - `PROMETHEUS_REMOTE_WRITE_URL`: The URL of the Prometheus remote_write-compatible server to send metrics to.
+  - `USERNAME`: The username to use for authentication to the remote_write API.
+  - `PASSWORD`: The password to use for authentication to the remote_write API.
+
+### Using private IP address:
+
+```
+discovery.linode "example" {
+    bearer_token = env("LINODE_TOKEN")
+    port = 8876
+}
+discovery.relabel "private_ips" {
+	targets = discovery.linode.example.targets
+	rule {
+    	source_labels = ["__meta_linode_private_ipv4"]
+    	replacement     = "[$1]:8876"
+    	target_label  = "__address__"
+  	}
+}
+prometheus.scrape "demo" {
+	targets    = discovery.relabel.private_ips.targets
+	forward_to = [...]
```

Comment:
I think we typically build out the full example in this section although it may be redundant in this case